diff --git a/server/index.ts b/server/index.ts
index 57584ea7..a05cf0c6 100644
--- a/server/index.ts
+++ b/server/index.ts
@@ -158,7 +158,7 @@ app
     if (settings.network.trustProxy) {
       server.enable('trust proxy');
     }
-    server.use(cookieParser(settings.clientId));
+    server.use(cookieParser(settings.sessionSecret));
     server.use(express.json());
     server.use(express.urlencoded({ extended: true }));
     server.use((req, _res, next) => {
@@ -224,6 +224,7 @@ app
       OpenApiValidator.middleware({
         apiSpec: API_SPEC_PATH,
         validateRequests: true,
+        validateSecurity: false,
       })
     );
     /**
